taxi/urls.py

```diff
@@ -42,10 +46,32 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/<int:pk>/driver_update/",
+        CarDriverUpdateView.as_view(),
+        name="car-driver-update",
     ),
+    path("drivers/",
+         DriverListView.as_view(),
+         name="driver-list"),
+
+    path("drivers/<int:pk>/",
+         DriverDetailView.as_view(),
+         name="driver-detail"),
+
+    path("drivers/create/",
+         DriverCreateView.as_view(),
+         name="driver_create"),
```

Comment:
The path name for `DriverCreateView` is `driver_create`, which uses an underscore. Consider changing it to `driver-create` to maintain consistency with other path names that use hyphens.